libs/island-ui/core/src/lib/ActionCard/ActionCard.tsx

```diff
@@ -237,7 +237,7 @@ export const ActionCard: React.FC<React.PropsWithChildren<ActionCardProps>> = ({
     }
 
     return (
-      <Box marginTop={2}>
+      <Box marginTop={2} marginRight={5}>
```

Comment:
Should this spacing be here? Would the parent be able to do the spacing instead of changing the core component?

Reply:
yes, should be set here for the progress meter to not fill up the entire line. Looks messy if the card button is visible
![Screenshot 2024-08-26 at 11 21 03](https://github.com/user-attachments/assets/d0ab9ea5-b813-448e-a889-8cf5d4ccf16e)
![Screenshot 2024-08-26 at 11 21 27](https://github.com/user-attachments/assets/825395de-463b-4c74-b942-944158503f45)